pkg/cluster/healthcheck.go

```diff
@@ -68,3 +68,8 @@ func (u *simpleUpgrader) HealthCheck(ctx context.Context, cs *api.OpenShiftManag
 	u.log.Info("checking admin console health")
 	return u.doHealthCheck(ctx, getHealthCheckHTTPClient(cs), "https://console."+cs.Properties.RouterProfiles[0].PublicSubdomain, 10*time.Second)
 }
+
+func (u *simpleUpgrader) WaitForHealthzStatusOk(ctx context.Context, cs *api.OpenShiftManagedCluster) error {
+	_, err := wait.ForHTTPStatusOk(ctx, u.log, u.rt, "https://"+cs.Properties.FQDN+"/healthz")
```

Comment:
can you think about whether it's worth removing `u.rt` at this point if the layering is better?  not sure either way - just have a think about it

Reply:
we *could*, but after trying it, it makes the plugin_test.go messy